src/models/DomainsReverseResolution.ts

```diff
@@ -0,0 +1,32 @@
+import { Column, Entity, Index, JoinColumn, ManyToOne, Unique } from 'typeorm';
+import { Matches, IsEnum } from 'class-validator';
+import { Domain, Model } from '.';
+import { Attributes, Blockchain } from '../types/common';
+import { ETHAddressRegex } from '../utils/ethersUtils';
+import { lowercaseTransformer } from '../database/transformers';
+
+@Entity({ name: 'domains_reverse_resolution' })
+@Unique(['domain', 'blockchain', 'networkId'])
+@Index(['domain', 'blockchain', 'networkId', 'reverseAddress'])
+export default class DomainsReverseResolution extends Model {
+  @Matches(ETHAddressRegex) // ensure that the address is lowercase to be consistent
+  @Column('text', { transformer: lowercaseTransformer })
```

Comment:
Sorry I should've explained it more clearly in my previous comment, but what I was envisioning was to have comments in the DB layer, something like
```suggestion
  @Column('text', { transformer: lowercaseTransformer, comment: 'unique index, the ethereum address that is configured for reverse resolution' })
```
so it would be beneficial for someone who operates the DB without checking the source code.

Reply:
Yes, This makes more sense. For some reason I assumed inline comments